mycore-base/src/main/java/org/mycore/common/config/MCRInstanceConfiguration.java

```diff
@@ -129,16 +129,20 @@ public static MCRInstanceConfiguration ofName(MCRInstanceName name, Map<String,
         return new MCRInstanceConfiguration(name, className, reducedProperties, properties);
     }
 
+    @SuppressWarnings("PMD.CollapsibleIfStatements")
+    //The condition was intentionally split into to warts, to make it to follow the thought-process.
```

Comment:
My original comment was indeed garbled up, should have been:

> The condition was intentionally split into two parts, to make it easier to follow the thought-process.
> The latter two checks only make sense in the case, where the first condition is met.

Reply:
The comment was corrected in the following commit.

Reply:
As far as I can tell, the comment still contains the unintelligible part "into to warts, to make it to follow the thought-process"